core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-edit-page-info/dot-edit-page-info.component.ts

```diff
@@ -11,21 +16,34 @@ import { ChangeDetectionStrategy, Component, Inject, Input } from '@angular/core
     selector: 'dot-edit-page-info',
     templateUrl: './dot-edit-page-info.component.html',
     styleUrls: ['./dot-edit-page-info.component.scss'],
-    changeDetection: ChangeDetectionStrategy.OnPush
+    changeDetection: ChangeDetectionStrategy.Default
```

Comment:
this can be removed when is set to default ?

Reply:
@hmoreras I think could be removed going to check.